clients/pkg/promtail/promtail.go

```diff
@@ -260,7 +260,7 @@ func (p *Promtail) watchConfig() {
 		level.Warn(p.logger).Log("msg", "disable watchConfig", "reason", "promtailServer cast fail")
 		return
 	}
-	level.Warn(p.logger).Log("msg", "enable watchConfig")
+	level.Debug(p.logger).Log("msg", "enable watchConfig")
```

Comment:
we can just move forward with your pr @diranged , can you make this line more descriptive, as well as the earlier ones with `disable watchConfig` please

I think they should be something like `"live reloading of promtails config is enabled/disabled"`